### DISCUSSION TITLE
blog/simple-authentication-approach-with-actix-web

### DISCUSSION THREAD

**Discussion by giscus** [geofmureithi/my_blog#1] [General]
# blog/simple-authentication-approach-with-actix-web

Learn how to painlesly apply authentication to your actix-web project

https://mureithi.me/blog/simple-authentication-approach-with-actix-web#comments-container

**Comment by geofmureithi** [comment]
NB: You can learn more about [extractors](https://actix.rs/docs/extractors/) on their site.
